Add link for Java workload destination guide (#1383)
MicrosoftDocs/azure-dev-docs: articles/java/spring-framework/deploy-spring-boot-java-app-on-kubernetes.md

# Deploy Spring Boot Application to the Azure Kubernetes Service

> [!NOTE]

> > For Spring Boot applications, we recommend using Azure Spring Apps. However, you can still choose to use Azure Kubernetes Service as a destination. See [Java Workload Destination Guidance](https://aka.ms/javadestinations) for advice.

Reviewer: ```suggestion
> For Spring Boot applications, we recommend using Azure Spring Apps. However, you can still choose to use Azure Kubernetes Service as a destination. For more information, see [Choose the right Azure services for your Java applications](../get-started/choose.md).
```
